ci/gpu/build.sh

```diff
@@ -38,7 +38,7 @@ $CXX --version
 
 logger "Setup new environment..."
 conda install -c rapidsai/label/cuda$CUDA_REL -c rapidsai-nightly/label/cuda$CUDA_REL -c nvidia/label/cuda$CUDA_REL -c conda-forge \
-    cudf=0.6* \
+    cudf=0.7* \
```

Comment:
In anticipation of 0.8, we might consider dropping the version constraint altogether or, at least, provide a `>=` constraint rather than strict equality.

Reply:
Makes sense. Will update. Need to triage the fact that the query now breaks due to not finding `local_dict` in the call environment. I was able to pass all of these tests locally in an environment different than the gpuci env, so will need to look into it.